dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype-mbeans.properties

```diff
@@ -109,3 +109,6 @@ plugin.trustedProxyEnable.desc=Enables trusted proxies to be used. When specifie
 
 plugin.trustedProxyGroup.name=Override the TrustedProxyGroup property
 plugin.trustedProxyGroup.desc=A comma-separated list of trusted proxies. When specified, this property overrides the value in the web server plug-in configuration file.
+
+plugin.ignoreAffinityRequests.name=Ignore affinity requests
+plugin.ignoreAffinityRequests.desc=Indicates whether affinity request's are ignored when tracking a servers runtime weight for round robin load balancing
```

Comment:
I think the description should be `Indicates whether affinity requests are ignored when tracking a server's runtime weight for round robin load balancing`, however it might make sense to wait for the ID/message review

Reply:
you are right, I put the apostrophe in the wrong place.